src/client/common/tooltips/generateContent.js

```diff
@@ -3,23 +3,32 @@ const classNames = require('classnames');
 const _ = require('lodash');
 const config = require('./config');
 
+//Define entry limit for trim mode
+const defaultEntryLimit = 3;
```

Comment:
these constants could probably be moved to the config.

Reply:
I have moved the constants to the config file.